### DISCUSSION TITLE
Slice transparency in 3D visualization

### DISCUSSION THREAD

**Discussion by reigo-nitsugua** [brainvisa/brainvisa.github.io#102] [Q&A]
Hello,

I have been using Anatomist for some time to make figures and I would like first to thank you for this software, it is so useful!

I have an issue with a 3D visualization feature. I would like to make a slice object transparent in places (where there is a value of 0 in the image) in order to see a mesh through it. To illustrate, in the image below, I would like to see the blue mesh through the part I have set to 0 on the slice.
![kEWIzoI](https://user-images.githubusercontent.com/103630499/164480533-319079d7-ed96-4687-9d43-c1cb912a5eab.png)

I tried to reproduce this [example](https://brainvisa.info/anatomist/user_doc/anatomist_manual2.html#example) but it seems to work only for a 2D view and not 3D.

Is it possible to do what I am trying to do? If so, could you please tell me how to do this?

Thank you,

**Comment by sapetnioc** [comment]
Well, I was not able to do what you tried to do. I am afraid that the only one who can answer is @denisri. He will be back next week.

**Comment by denisri** [comment]
Hi, sorry for the delay...
Well, first, **there are no 2D views in Anatomist**: all are 3D, it's just a matter of where the camera is and how slices are oriented. So something working in 2D should also work in 3D.
I think actually everything is explained in the link you point in the docs, it's just the end of the explanation you have missed I guess: to make a slice actually transparent (and not only its texture), you have to set its texturing mode to something different from the default `Decal` mode: using `Replace` or `Geometric` will assign the mesh opacity to the texture opacity, thus actually making it transparent when used with a slice itself using a [semi]transparent palette.
In brief, in your example:
- set a palette with transparency on the image, exactly as you did: `B-W LINEAR-transparent` is OK
- open the texturing properties on the volume, and set the "mapping mode" to `Replace`

Now the slice is actually transparent.
